A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js

```diff
@@ -2837,7 +2837,7 @@ class Warnings extends NavSystemElement {
     landingGearCallback() {
         const gear = !SimVar.GetSimVarValue("IS GEAR RETRACTABLE", "Boolean") || SimVar.GetSimVarValue("GEAR HANDLE POSITION", "Boolean");
         const throttle = SimVar.GetSimVarValue("L:A32NX_AUTOTHRUST_TLA:1", "number");
-        const flaps = SimVar.GetSimVarValue("L:A32NX_FLAPS_HANDLE_INDEX", "number");
+        contst flaps = SimVar.GetSimVarValue("L:A32NX_FLAPS_HANDLE_INDEX", "number");
```

Comment:
huh?

Reply:
forget it...I have overseen it. I do not understand why there was in any case a conflict on that line. Using the git function was big mess.